avanza/avanza.py

```diff
@@ -1670,6 +1671,67 @@ def place_order_sell_fund(
         )
 
 
+    def place_stop_loss_order(
+        self,
+        parent_stop_loss_id: str,
+        account_id: str,
+        order_book_id: str,
+        stop_loss_trigger: Type[StopLossTrigger],
+        stop_loss_order_event: Type[StopLossOrderEvent],
```

Comment:
Shouldn't these just be StopLossTrigger and StopLossOrderEvent, without being wrapped in a `Type[]`?

https://docs.python.org/3/library/typing.html#typing.Type

Reply:
Probably should, good catch. I'll update the PR.